public/_data/repositories/core.yml

```diff
@@ -0,0 +1,39 @@
+section:
+  description: These are the primary repositories for Conjur Open Source and its SDK.
+  categories:
+  - name: Conjur OSS Core
+    description: Conjur OSS Server and Deployment Tools
+    repos:
+      - name: cyberark/conjur
+        url: https://github.com/cyberark/conjur
+        description: Conjur OSS server. Conjur comes built-in with custom authenticators
+          for Kubernetes, OpenShift, AWS IAM, OIDC, and more.
+        thumb: CyberArk_Conjur_logo.svg
+        #image-alt:
```

Comment:
[NIT] In this file, there's a thumb with a commented out alt-text; in delivery.yml, both the thumb and alt are commented out; and integrations.yml has neither. Not sure if the two files that have the thumb and image-alt are meant to both be commented out. If the intent is to have the image here, it would be nice to specify the Alt text too, and then probably remove the comment in the delivery.yml file. If they're both meant to be commented out, seems like all the comment lines should be removed.

Reply:
I wanted to leave open the option of adding images later - right now the thumb and image-alt fields aren't included, and I suppose they're left there as breadcrumbs for if this functionality gets added later.

I'm going to leave it for now, since this all might get migrated anyway, but may clean it up later if I don't add this functionality and this site ends up sticking around.